benchmarks/functional_autograd_benchmark/compare.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+from collections import defaultdict
+
+from utils import to_markdown_table, from_markdown_table
+
+def main():
+    parser = argparse.ArgumentParser("Main script to compare results from the benchmarks")
+    parser.add_argument("--before", type=str, default="before.txt", help="Text file containing the times to use as base")
+    parser.add_argument("--after", type=str, default="after.txt", help="Text file containing the times to use as new version")
+    parser.add_argument("--output", type=str, default="", help="Text file where to write the output")
+    args = parser.parse_args()
+
+    with open(args.before, "r") as f:
+        content = f.read()
+    res_before = from_markdown_table(content)
+
+    with open(args.after, "r") as f:
+        content = f.read()
+    res_after = from_markdown_table(content)
+
+    diff = defaultdict(defaultdict)
+    for model in res_before:
+        for task in res_before[model]:
+            mean_before, var_before = res_before[model][task]
+            if task not in res_after[model]:
+                diff[model][task] = (-1, mean_before, var_before, -1, -1)
+            else:
+                mean_after, var_after = res_after[model][task]
+                diff[model][task] = (mean_before / mean_after, mean_before, var_before, mean_after, var_after)
+    for model in res_after:
+        for task in res_after[model]:
+            if task not in res_before[model]:
+                mean_after, var_after = res_after[model][task]
+                diff[model][task] = (-1, -1, -1, mean_after, var_after)
```

Comment:
Is it better to use None as the indicator value? Some users might interpret the -1 as "something went wrong" (whenever I saw negative values in fastrnns.bench, that was a sign to me that there was a bug, but only because they were negative floating-point values that looked absurd.)

Reply:
Very good point! Thanks!